### DISCUSSION TITLE
The sidebar doesn't have the theme applied, is this normal?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [GradienceTeam/Gradience#860] [Q&A] [answered]
![image](https://github.com/GradienceTeam/Gradience/assets/98482469/0e5f57b0-0f20-4b2f-a7af-fc0673485ddc)

**Comment by daudix** [comment, marked as answer]
See #828

**Comment by Heus-Sueh** [comment]
https://github.com/GradienceTeam/Gradience/issues/828#issuecomment-1759547952
